### ISSUE TITLE
PHP Warning:  Trying to access array offset on value of type bool

### ISSUE THREAD

**Issue by robindevitt** [humanmade/tachyon#154]
During some testing for Altis V13 ( testing on V12 too ) there are the following PHP warnings:
```php
PHP Warning:  Trying to access array offset on value of type bool in /usr/src/app/vendor/humanmade/tachyon-plugin/inc/class-tachyon.php on line 631
PHP Warning:  Trying to access array offset on value of type bool in /usr/src/app/vendor/humanmade/tachyon-plugin/inc/class-tachyon.php on line 632
PHP Warning:  Trying to access array offset on value of type bool in /usr/src/app/vendor/humanmade/tachyon-plugin/inc/class-tachyon.php on line 635
PHP Warning:  Trying to access array offset on value of type bool in /usr/src/app/vendor/humanmade/tachyon-plugin/inc/class-tachyon.php on line 635
PHP Warning:  Trying to access array offset on value of type bool in /usr/src/app/vendor/humanmade/tachyon-plugin/inc/class-tachyon.php on line 653
PHP Warning:  Trying to access array offset on value of type bool in /usr/src/app/vendor/humanmade/tachyon-plugin/inc/class-tachyon.php on line 653
```

#### Steps to reproduce
1. Ensure your install is running either Altis V13 or Altis V12. ( [Getting started ](https://docs.altis-dxp.com/v13/getting-started/) or [Upgrade guide](https://docs.altis-dxp.com/v13/guides/upgrading/) )
2. Upload a new image to the media library.
3. Observe the php logs. `composer server logs php` for the above mentioned warnings. ( [viewing logs](https://docs.altis-dxp.com/v13/local-server/viewing-logs/) ) 

This only occurs once when an image is initially uploaded.

#### Acceptance criteria
- [ ] Upload an image and see no php warnings.

**Comment by jerico** [comment]
Testing getting blocked by a different error when uploading

```
Fatal error: Uncaught Exception: Error executing "ListObjects" on "https://s3-tachyon-test.altis.dev/s3-tachyon-test//?delimiter=%2F&prefix=uploads%2F2022%2F11%2Fa&encoding-type=url"; AWS HTTP error: Client error: `GET https://s3-tachyon-test.altis.dev/s3-tachyon-test//?delimiter=%2F&prefix=uploads%2F2022%2F11%2Fa&encoding-type=url` resulted in a `400 Bad Request` response:
<?xml version="1.0" encoding="UTF-8"?>
<Error><Code>XMinioInvalidObjectName</Code><Message>Object name contains unsuppor (truncated...)
XMinioInvalidObjectName (client): Object name contains unsupported characters. - <?xml version="1.0" encoding="UTF-8"?>
<Error><Code>XMinioInvalidObjectName</Code><Message>Object name contains unsupported characters.</Message><BucketName>s3-tachyon-test</BucketName><Resource>/s3-tachyon-test//</Resource><Region>us-east-1</Region><RequestId>17284A7BBAE69FAE</RequestId><HostId>67513999-84ff-4e9e-a421-925f5f9e759f</HostId></Error>
in /usr/src/app/vendor/aws/aws-sdk-php/src/WrappedHttpHandler.php on line 195
```

Investigating

**Comment by jerico** [comment]
Seems like a known issue: https://github.com/humanmade/altis-cloud/issues/700

**Comment by jerico** [comment]
I spent more time on this to try identify the issue. Posted my findings to the other ticket https://github.com/humanmade/altis-cloud/issues/700

TLDR: Uploading works after if `local-server` is rolled back to to v10. I did not find the change the affected this functionality yet.

**Comment by jerico** [comment]
Confirmed that https://github.com/humanmade/tachyon-plugin/commit/82028419b61201eb952a9696dde92cebe0d26e6c fixed this issue.

**Comment by mikelittle** [comment]
Looks like this can be closed based on confirmation of the fix.
